### DISCUSSION TITLE
Please help. Thank you

### DISCUSSION THREAD

**Discussion by RiseToTop** [gitbrent/PptxGenJS#1303] [Q&A]
The pptx file I exported using PptxGenJS cannot be opened in powerpoint. powerpoint is the 2019 version, while PptxGenJS is the latest version

Here is the error message when you open powerPoint

Microsoft PowerPoint

PowerPoint found a problem with the content in C:/Users\lihao\Downloads\phone-drop-2.pptx.

PowerPoint can try to fix this presentation.

If you trust the source of this presentation, click Repair.

**Comment by Dreamer-Somebody** [comment]
If this can help: https://github.com/gitbrent/PptxGenJS/pull/1083

**Comment by Dreamer-Somebody** [comment]
You can edit the source code through pnpm patch and patch it to solve it temporarily
